ext/reflection/php_reflection.stub.php

```diff
@@ -700,7 +699,7 @@ public function hasCase(string $name): bool {}
 
     public function getCase(string $name): ReflectionEnumUnitCase {}
 
-    /** @return ReflectionEnumUnitCase[] */
+    /** @tentative-return-type */
```

Comment:
Not needed.

Reply:
I fixed this while applying the commit to themaster branch!